Readme.md

```diff
@@ -5,7 +5,18 @@
 node-prune is a small tool to prune unnecessary files from ./node_modules, such as markdown, typescript source files, and so on. Primarily built for [Up](https://github.com/apex/up) which lets you deploy serverless web applications in seconds.
 
 ## Installation
+If Go is already installed check check if your are using latest version, comparing to https://github.com/golang/go/releases, type this command to check your local Go version 
```

Comment:
Spelling: _you_ are.

Reply:
Two instances of _check_.

Reply:
I would lean on pointing to the `golang` docs on how to install and update your package instead of trying to maintain it here.  At some point its going to be out of sync!